src/backend/database_models/blacklist.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy import Index, String
+from sqlalchemy.orm import Mapped, mapped_column
+
+from backend.database_models.base import Base
+
+
+class Blacklist(Base):
+    """
+    Table that contains the list of JWT access tokens that are blacklisted during logout.
+    """
+
+    __tablename__ = "blacklist"
+
+    token_id: Mapped[str] = mapped_column(String)
```

Comment:
Is this field used to store blacklisted tokens? I think it would be better to name it 'token'.

Reply:
This would store the ID of the token rather than the payload itself